tests/aio-create.sh

```diff
@@ -219,10 +223,21 @@ pushd /opt/openstack-ansible
 
   # Pin python-ldap so it stops breaking everything
   echo "python-ldap<3;python_version=='2.7'" >> /opt/openstack-ansible/global-requirement-pins.txt
+  # Fixing missing netbase packages
+  sudo apt-get -o Dpkg::Options::="--force-confmiss" install --reinstall netbase
 
   # Disable the sec role
   disable_security_role
 
   # Setup an AIO
   sudo -H --preserve-env ./scripts/gate-check-commit.sh
+
+  # # NOTE(tonytan4ever): Help debugging the apt issue.
+  # Implement debug output for apt so that we can see more information
+  # about whether the 'Acquire-by-hash' feature is being used, and what
+  # might be causing it to fall back to the old style.
+  # This config file should be copied into containers by the lxc_hosts
+  # role.
+  # ansible hosts -m lineinfile -a 'create=yes dest=/etc/apt/apt.conf.d/99debug line="Debug::Acquire::http \"true\";"'
```

Comment:
This is commented, so it does nothing. Also it'd be more useful to have the debug output in the AIO during the initial build, not after it's built - so this would be better to do, before gate-check-commit executes: https://github.com/rcbops/rpc-openstack/commit/b0215bbf69c9b6eaf148e332a32564a0101c88de#diff-93518fda10b0403d3c5c20b4df4740a6